spec/cloudinary_spec.rb

```diff
@@ -28,5 +28,16 @@
       Cloudinary::config_from_url ENV["CLOUDINARY_URL"]
       expect(Cloudinary::config.foo.bar).to eq 'value'
     end
+    it "should raise an exception if the CLOUDINARY_URL doesn't start with 'cloudinary://'" do
+      invalid_cloudinary_urls = [
```

Comment:
Please use input values from the reference implementation

Reply:
Used the reference implementation values but separated the test cases as they throw different types of Exceptions.